### PR TITLE
Add 2nd blog for inkind sponsors

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
CC: @vipulgupta2048 @raukadah @anistark

### CODE REVIEW

**Review by raukadah — APPROVED** [review]

File: content/blog/2021/announcing-inkind-sponsors-second.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Title: Announcing some of our InKind sponsrs for PyCon India 2021 - Packt, Innovaccer, Viprush Technologies, and Toyota Connected India
```
Comment (vipulgupta2048) [inline content/blog/2021/announcing-inkind-sponsors-second.md:1]:
```suggestion
Title: Announcing Packt, Innovaccer, Viprush Technologies, and Toyota Connected India as In-Kind sponsors for PyCon India 2021 (Part 1)
```

**Review by vipulgupta2048 — COMMENTED** [review]
InKind like this looks weird. Until it's followed like everywhere. 
How about `In-kind`?

File: content/blog/2021/announcing-inkind-sponsors-second.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2021/announcing-inkind-sponsors-second.md:1]:
```suggestion
Title: Announcing Packt, Innovaccer, Viprush Technologies, and Toyota Connected India as our In-Kind sponsors for PyCon India 2021 (Part 2)
```

**Comment by raukadah** [comment]
@palnabarun We need to deploy it today i.e Sep 6th 2021

**Review by raukadah — APPROVED** [review]

**Pull request merged**